source/guides/network/supernode-enrollment.rst

```diff
@@ -179,6 +223,21 @@ You can **validate your node** by checking that all services are running properl
 
 * `http://localhost:3000/chain/info <http://localhost:3000/chain/info>`__: Node's connection to the network.
 * `http://localhost:3000/node/info <http://localhost:3000/node/info>`__: Node's health.
-* `https://localhost:7880/metadata <https://localhost:7880/metadata>`__: Agent's report.
+* `https://localhost:7881/metadata <https://localhost:7881/metadata>`__: Agent's report.
```

Comment:
Instead of this one should we put something like this ⇒ `curl -k https://localhost:7881/metadata`
Because the browser will complain about TLS verification and won't allow users to see the result .
Check these please:
https://ec2-44-227-13-220.us-west-2.compute.amazonaws.com:7881/metadata
`curl -k https://ec2-44-227-13-220.us-west-2.compute.amazonaws.com:7881/metadata`

Reply:
I already added a note in this regard in the next paragraph. I'll make it clearer.